TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/kafka/KafkaProtobufConsumerTest.java

```diff
@@ -127,7 +127,6 @@ public void testSupplyTrafficWithUnformattedMessages() {
     @Test
     public void testBuildPropertiesBaseCase() throws IOException {
         Properties props = KafkaProtobufConsumer.buildKafkaProperties("brokers", "groupId", false, null);
-        Assertions.assertEquals(5, props.size());
```

Comment:
Just curious - why did these need to go away?

Reply:
Mainly removing these as they are not maintainable checks. Anytime we add or remove kakfa settings this would need to change